### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by harshkhandeparkar**

### CODE REVIEW

File: frontend/src/styles/styles.scss [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+// :root {
+//   --accent-color: hsl(217, 80%, 40%);
+//   --accent-color-darker: hsl(217, 80%, 30%);
+//   --bg-color: #e8e5e5;
+//   --surface-1: #fff;
+//   --fg-color: #333;
+//   --fg-inverse: #fff;
+// }
+
+:root {
```
Comment (harshkhandeparkar) [inline frontend/src/styles/styles.scss:11]: Why are these changes showing up here?

Comment (shikharish) [reply frontend/src/styles/styles.scss]: fixed

File: README.md [hunk 2]
```diff
@@ -40,117 +32,33 @@
...
 - [Contact](#contact)
 - [Additional documentation](#additional-documentation)
 
 </details>
 
-<!-- ABOUT THE PROJECT -->
-
 ## About The Project
+IQPS is a platform for searching and uploading past year question papers for IIT Kharagpur students.
+> **NOTE:** IQPS is currently in active development. The beta version is deployed at [beta.qp.metakgp.org](https://beta.qp.metakgp.org/)
```
Comment (harshkhandeparkar) [inline README.md:45]: Add Slack link as well. "In active development. Get involved at our [Slack]"

Comment (shikharish) [reply README.md:45]: > **NOTE:** Currently in active development. The beta version is deployed at [beta.qp.metakgp.org](https://beta.qp.metakgp.org/). Get involved at our [Slack](https://slack.metakgp.org/).

This okay?

Comment (harshkhandeparkar) [reply README.md:45]: I think beta version link need not be specified separately as it is the only link.

Comment (shikharish) [reply README.md:45]: I feel that it should be explicitly mentioned that it is in beta right now.

Comment (harshkhandeparkar) [reply README.md:45]: I meant the link is not required, keep the active development text.

**Comment by chirag-ghosh** [comment]
@harshkhandeparkar updates please

**Comment by harshkhandeparkar** [comment]
> @harshkhandeparkar updates please

Was I supposed to do something?

**Comment by harshkhandeparkar** [comment]
This is so old I lost context lol

**Comment by chirag-ghosh** [comment]
> > @harshkhandeparkar updates please
> 
> Was I supposed to do something?

image
Work khatam karke updraft karo sir

**Pull request merged**